vg/build/Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM ubuntu:14.04
+FROM ubuntu:15.10
 
 MAINTAINER Charles Markello, cmarkell@ucsc.edu
 
 # Install vg dependencies
-RUN apt-get update \
-	&& apt-get update && apt-get install -y \
-	build-essential \
-	gcc-4.9 \
-	g++-4.9 \
-	pkg-config \
-	sudo \
-	git \
-	make \
-	protobuf-compiler \
-	libprotoc-dev \
-	libjansson-dev \
-	libbz2-dev \
-	libncurses5-dev \
-	automake libtool jq samtools curl unzip redland-utils \
-	librdf-dev cmake pkg-config wget bc gtk-doc-tools raptor2-utils rasqal-utils bison flex
+RUN apt-get update && \
+	apt-get update &&\
```

Comment:
Just noticing that now: Why run update twice?

Is this change part of the fix? If not, can we drop it or put it into a separate commit?